packages/gamut-labs/src/AppHeader/Search/SearchButton.tsx

```diff
@@ -0,0 +1,38 @@
+import { SearchIcon } from '@codecademy/gamut-icons';
+import { system } from '@codecademy/gamut-styles';
+import styled from '@emotion/styled';
+import React from 'react';
+
+const SearchIconButton = styled.button(
+  system.css({
+    backgroundColor: 'transparent',
+    pt: 8,
+    border: 'transparent',
+    '&:hover': {
+      color: 'hyper',
+      cursor: 'pointer',
+    },
+    '&:focus': {
+      outline: 'none',
+    },
+    '&:focus-visible': {
+      outline: 'navy',
+    },
```

Comment:
you should be able to combine these like: 

```suggestion
    '&:focus, &:focus-visible': {
      outline: 'none',
    },
```

Reply:
> `outline: 'navy',`

I cannot 😄

Reply:
lol i misread, oop!